README.md

```diff
@@ -90,13 +101,23 @@ def mystery5(s)
 end
 ```
 
-- What is mystery5("hi")?
-- What is mystery5("")?
-- What is mystery5("Hi, there!")?
+- What is mystery5("hi")? "**"
+- What is mystery5("")? ""
+- What is mystery5("Hi, there!")? "**********"
 - _Added Fun: How could we make only alphabetic characters to be changed to stars?_
+  ```ruby
+  def mystery5(s)
+    if s.length == 0
+      return ""
+    else
+      s[0] = "*" if s[0].match(/[a-zA-Z]/)
+      return s[0] + mystery5(s[1..-1])
```

Comment:
This looks like it works, but it modifies "s". I'm not sure what exactly this does in Ruby, but you might not want to do this always. Alternatively, you could have done:

```ruby
def mystery5(s)
  if s.length == 0
    return ""
  elsif s[0].match(/[a-zA-Z]/)
    return "*" + mystery5(s[1..-1])
  else
    return s[0] + mystery5(s[1..-1])
  end
end
```